main.py

```diff
@@ -1,10 +1,67 @@
+import os
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    players = file_data("players.json")
```

Comment:
Ensure that the 'players.json' file exists in the correct directory, or handle the potential FileNotFoundError exception in the 'file_data' function.